packages/edit-site/src/components/sidebar-edit-mode/global-styles-sidebar.js

```diff
@@ -17,25 +18,36 @@ import { GlobalStylesMenuSlot } from '../global-styles/ui';
 import { unlock } from '../../private-apis';
 
 export default function GlobalStylesSidebar() {
-	const { editorMode, editorCanvasView } = useSelect( ( select ) => {
-		return {
-			editorMode: select( editSiteStore ).getEditorMode(),
-			editorCanvasView: unlock(
-				select( editSiteStore )
-			).getEditorCanvasContainerView(),
-		};
-	}, [] );
+	const { shouldDisableStyleBook, isStyleBookOpened } = useSelect(
+		( select ) => {
+			const { getActiveComplementaryArea } = select( interfaceStore );
+			const _isVisualEditorMode =
+				'visual' === select( editSiteStore ).getEditorMode();
+
+			return {
+				isVisualEditorMode: _isVisualEditorMode,
```

Comment:
Is this being used anywhere? It doesn't look like it needs to be returned in the object at first glance 🤔

Reply:
Ugh. Thanks. Will fix.